ISSUES.md

```diff
@@ -0,0 +1,27 @@
+# Known External Issues
```

Comment:
Please add a little something that for issues that can/should be fixed, creating a Github issue is the right way to go. This is *only* as a reference for issues that are known not to be solved quickly (as in: before Supercon).

Reply:
The opening text is now more succinct and include the explicit preference for tracking problems via Github issue.